java/jdbc/ConnectionSamples/ADBQuickStart.java

```diff
@@ -0,0 +1,149 @@
+ 
+ (a) Set the connection factory class name to 
+ oracle.jdbc.pool.OracleDataSource before getting a connection.   
+ (b) Set the connection pool properties(e.g.,minPoolSize, maxPoolSize). 
+ (c) Get the connection and perform some database operations. 
+ For a quick test, the sample retrieves 20 records from the Sales History (SH) schema 
+ that is accessible to any DB users on autonomous Database.  
+ 
+ Step 1: Enter the Database details DB_URL and DB_USER. 
+ You will need to enter the DB_PASSWORD of your Autonomous Database through console
+ while running the sample.  
+ Step 2: Download the latest Oracle JDBC driver(ojdbc8.jar) and UCP (ucp.jar) 
+ along with oraclepki.jar, osdt_core.jar and osdt_cert.jar and add to your classpath.  
+ Refer to https://www.oracle.com/database/technologies/maven-central-guide.html               
+ Step 3: Compile and Run the sample. 
+ 
+ SH Schema: 
+ This sample uses the Sales History (SH) sample schema. SH is a data set suited for 
+ online transaction processing operations. The Star Schema Benchmark (SSB) sample schema 
+ is available for data warehousing operations. Both schemas are available 
+ with your shared ADB instance and do not count towards your storage. 
+ ou can use any ADB user account to access these schemas.
+ 
+ NOTES
+ Use JDK 1.8 and above 
+  
+ MODIFIED    (MM/DD/YY)
+ nbsundar    11/09/2020 - Creation 
+ */
+package com.oracle.jdbctest;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Scanner;
+
+import oracle.ucp.jdbc.PoolDataSourceFactory;
+import oracle.ucp.jdbc.PoolDataSource;
+
+public class ADBQuickStart {
+ 
+  // Make sure to have Oracle JDBC release 18c or higher 
+  // to pass TNS_ADMIN as part of a connection URL.
+  // TNS_ADMIN - Should be the path where the client credentials zip file is downloaded. 
+  // wallet_dbname - This is the TNS Alias that you can get from tnsnames.ora.
+  final static String DB_URL="jdbc:oracle:thin:@wallet_dbname?TNS_ADMIN=/Users/test/wallet_dbname/";
+  // Update the Database Username to point to your Autonomous Database. For a quick testing, you can 
+  // use "admin" user. 
+  final static String DB_USER = "admin";
+  // For security reasons, you will need to provide database password through console.  
+  static String DB_PASSWORD ;
+  final static String CONN_FACTORY_CLASS_NAME="oracle.jdbc.pool.OracleDataSource";
+
+  /*
+   * The sample demonstrates UCP as client side connection pool.
+   */
+  public static void main(String args[]) throws Exception {
+    // For security purposes, you must enter the password through the console 
+    try {
+      Scanner scanner = new Scanner(System.in);
+      System.out.print("Enter the password for Autonomous Database: ");
+      DB_PASSWORD = scanner.nextLine();
+    }
+    catch (Exception e) {
+       System.out.println("ADBQuickStart - Exception occurred : " + e.getMessage());
```

Comment:
If an exception occurs here then the password is likely bad. The application cannot continue. Exit with value `1` or higher.

Reply:
Done